bookmarklet.js

```diff
@@ -71,7 +81,7 @@ javascript:(
                             }
 
                             /* Do nothing if there is no object */
-                            obj && copyToClipboard(JSON.stringify(obj, null, 4));
+                            obj && copyToClipboard(JSON.stringify(obj, null, 4)) && downloadFile(JSON.stringify(obj, null, 4));
```

Comment:
@Frozemint 

While you're here, could you try assigning the value `4` in the call to `JSON.stringify` to a variable? I'm not sure what it does here 🤔

Reply:
stringify pretty prints the JSON object, because by default the JSON prints on a single line. I think the 4 indicates how many spaces are used for indention (from top of my head)

Reply:
Oh I meant I didn't know what the number `4` does 😅

Would be pretty embarrassing if I didn't know what `JSON.stringify` did. 

> I think the 4 indicates how many spaces are used for indention

That sounds good to me, maybe you can put it in a constant like

```JavaScript
const defaultIndentation = 4;
```

I think we have a bunch of constants declared at the start of the function so they could go there.

Reply:
no worries, I have no idea how to finish 310 c0 either
check out https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/JSON/stringify if u want to learn more